kitchen/project.clj

```diff
@@ -10,7 +10,7 @@
 ;;
 (defproject kitchen "0.1.0-SNAPSHOT"
   :dependencies [^{:voom {:repo "https://github.com/twosigma/jet.git" :branch "waiter-patch"}}
-                 [cc.qbits/jet "0.7.10-20180124_201515-g857338f"]
```

Comment:
Does Kitchen need our custom-patched Jet, or could it use the official JAR?

Reply:
At the very least, we use the websocket support which has had some changes.